qiita_db/analysis.py

```diff
@@ -897,16 +775,17 @@ def build_files(self,
                         dup_samples = dup_samples | s
 
             self._build_mapping_file(samples, rename_dup_samples)
-            self._build_biom_tables(grouped_samples, rarefaction_depth,
-                                    rename_dup_samples)
+            biom_files = self._build_biom_tables(
+                grouped_samples, rename_dup_samples)
 
-    def _build_biom_tables(self, grouped_samples, rarefaction_depth=None,
-                           rename_dup_samples=False):
+            return biom_files
+
+    def _build_biom_tables(self, grouped_samples, rename_dup_samples=False):
```

Comment:
Note that this doesn't take into account grouping by processing parameter, which will merge deblur and close ref tables, fine to leave as is if it will be addressed in a future pr.